src/Facility.ConformanceApi/Testing/TestExternalDto.cs

```diff
@@ -0,0 +1,82 @@
+// <auto-generated>
+// DO NOT EDIT: generated by fsdgencsharp
```

Comment:
These files should be generated, but I don't see the FSD for the external types.

Also, the conformance tool would need to be able to output that FSD, just as it does the primary FSD. Should we demonstrate support for external types in `EdgeCases` and leave the conformance tests alone for now?

Reply:
I see how it makes sense to have those external test objects generated by an FSD as well. I will try to utilize `EdgeCases` instead of the conformance tests.

Reply:
I removed the extern types from the conformance tests and added a simple case to `EdgeCases`. Should there be anything additional in in the `GenerateEdgeCases` test method? Or is it just verifying that the code generation succeeds?